### PR TITLE
initial version of Azure Powershell swagger ci

### PR DESCRIPTION
**Pull request by dolauli**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: tools/SwaggerCI/gitignoreconf [hunk 1]
```diff
@@ -0,0 +1,5 @@
+bin
```
Comment (PhoenixHe-msft) [inline tools/SwaggerCI/gitignoreconf:1]: What's this file for?

File: tools/SwaggerCI/init.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -i
+
+# This script is created to set up the env to generate Azure PowerShell modules based on swagger. 
+
+# Installed required packages
+printenv
```
Comment (PhoenixHe-msft) [inline tools/SwaggerCI/init.sh:6]: Remove this line

File: tools/SwaggerCI/init.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+sudo apt-get update \
```
Comment (PhoenixHe-msft) [inline tools/SwaggerCI/init.sh:7]: Suggest to use https://www.npmjs.com/package/n to reduce noise and speedup.

N_PREFIX=$HOME/.n
PATH=$N_PREFIX/bin:$PATH
npx n 14.15.5

export PATH and N_PREFIX then

**Review by PhoenixHe-msft — CHANGES_REQUESTED** [review]

File: tools/SwaggerCI/gitignoreconf (hunk 1 above)
Comment (dolauli) [reply tools/SwaggerCI/gitignoreconf:1]: This file is used to override the autorest generated .gitignore file. We want to commit the generated code, while the generated one will ignore it.

File: tools/SwaggerCI/init.sh (hunk 3 above)
Comment (dolauli) [reply tools/SwaggerCI/init.sh:7]: Thanks for recommendation. Currently we use NVM in autorest.powershell projects across different OS. I will try to play with N for a while and then consider whether to replace NVM with N in the future.

File: tools/SwaggerCI/init.sh (hunk 2 above)
Comment (dolauli) [reply tools/SwaggerCI/init.sh]: Will do.

**Review by PhoenixHe-msft — APPROVED** [review]

**Pull request merged**